Scripts/travel_iteration.py

```diff
@@ -281,10 +281,7 @@ def assign_base_demand(self,
         mtx = self.ass_model.beeline_dist
         idx = numpy.where(numpy.isin(self.zdata_forecast.zone_numbers, self.zone_numbers))[0]
         mtx = mtx[idx[:, None], idx]
```

Comment:
This is not working as intended, because the  order of `self.zdata_forecast.zone_numbers` and `self.zone_numbers` is wrong. What it does now is actually equal to the much simpler:
```
idx = len(self.zdata_forecast.zone_numbers)
mtx = mtx[:idx, :idx]
```

I think that what you wanted to achieve is best expressed like this, because it is unnecessary to transform the boolean index into an integer index:

```
idx = numpy.isin(self.zone_numbers, self.zdata_forecast.zone_numbers)
mtx = mtx[numpy.ix_(idx, idx)]
```

Reply:
We could do it like that.